### PR TITLE
Updated /learning-objects to include optional parameter 'assigned'

### PR DESCRIPTION
**Pull request by StarChaser44**
Added new optional parameter to /learning-objects to include the new optional assigned parameter

### CODE REVIEW

**Comment by StarChaser44** [comment]
Screen Shot 2021-12-10 at 5 36 16 PM

**Review by JosueMolinaMorales — APPROVED** [review]

File: src/modules/learning-object-service/objects/ObjectsController.ts [hunk 1]
```diff
@@ -323,7 +323,7 @@ export class ObjectsController implements Controller {
 
     /**
      * @swagger
-     * /learning-objects:
+     * /learning-objects?assigned:
```
Comment (pzalep1) [inline src/modules/learning-object-service/objects/ObjectsController.ts:326]: I don't think we need to add the query parameter here. The comments below for the docs are enough.

Comment (EVogel1999) [reply src/modules/learning-object-service/objects/ObjectsController.ts:326]: ^

**Review by EVogel1999 — COMMENTED** [review]
Other then Paige's comment this looks good!

**Review by mfranz1 — APPROVED** [review]

**Review by jouypo23 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Pull request merged**